src/components/App.js

```diff
@@ -1,25 +1,27 @@
-import logo from '../assets/logo.svg';
 import '../styles/App.css';
+import { ChakraProvider } from '@chakra-ui/react';
+import { Container } from '@chakra-ui/react';
+import SignUp from './SignUp/SignUp';
+import LogIn from './LogIn/LogIn';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+	return (
+		<ChakraProvider>
+			<Container maxWidth='container.xl' p={0}>
+				<Router>
+					<Switch>
+						<Route path='/' exact>
```

Comment:
Can you move signup page to '/auth/signup'. Want to keep '/' for main page

Reply:
@zealptl sure thing!